stable/rabbitmq/templates/svc.yaml

```diff
@@ -14,13 +14,13 @@ spec:
     port: 4369
     targetPort: epmd
   - name: amqp
-    port: {{ default "5672" .Values.rabbitmqNodePort }}
+    port: {{ default "5672" .Values.rabbitmq.nodePort }}
```

Comment:
more defaults

Reply:
in my defence, they were there before :) - changed